### ISSUE TITLE
UAF 1.1: FASM-API-Android-1 P-2 fails to retrieve authenticatorIndex

### ISSUE THREAD

**Issue by mlourie** [fido-alliance/conformance-tool-issues#36]
We get request from conformance tool:

```
{"args":{"appID":"android:apk-key-hash:LK84ci82ruA6u8SyF26cghsVwZg="},"asmVersion":{"major":1,"minor":0},"requestType":"Deregister"}
```

Expected result: success
Actual result: fails

Incoming request is invalid per UAF specification: args must be set to an object of type DeregisterIn 

```
DeregisterIn {
required DOMString appID;
required DOMString keyID;
};
```

**Comment by herrjemand** [comment]
This test same as P-5/P-4/P-3 are failing because of the failing P-2, since is the one that is getting authenticatorIndex.

**Comment by herrjemand** [comment]
Ref #33 #34 #35

**Comment by herrjemand** [comment]
Closed in as affected by #30

**Comment by herrjemand** [comment]
@mlourie Added additional checks for the P-3/P-4/P-5/P-6 to clarify this issue
